src/domains/citizen-auth-common/99_locals.tf

```diff
@@ -36,4 +36,18 @@ locals {
 
   apim_itn_name                = "${local.product}-${local.itn_location_short}-apim-01"
   apim_itn_resource_group_name = "${local.product}-${local.itn_location_short}-common-rg-01"
+
+  prefix    = "io"
+  env_short = "p"
+  # domain          = "citizenauth"
+  app_name        = "citizenauth"
+  instance_number = "01"
```

Comment:
I think it is not necessary to have these locals since we have the itn_environment dictionary
```suggestion
```